onnxruntime/contrib_ops/cpu/tokenizer.cc

```diff
@@ -56,177 +52,12 @@ ONNX_CPU_OPERATOR_TYPED_MS_KERNEL(
     contrib::Tokenizer);
 
 namespace tokenizer_details {
-
 const char start_text = 0x2;
 const char end_text = 0x3;
-
-const std::string conv_error("Conversion Error");
```

Comment:
Se we drop the version with separators? Not possible to keep both?

Reply:
Not sure I understand the comment.

---
In reply to: [276860817](https://github.com/Microsoft/onnxruntime/pull/857#discussion_r276860817) [](ancestors = 276860817)

Reply:
Removing the behaviour with separators breaks backward compatibility. Converters in skl2onnx will have to be updated unless we have a version mechanism.

Reply:
I agree. We want to keep separators and preserve backward compatibility. This changes does it.

---
In reply to: [277124603](https://github.com/Microsoft/onnxruntime/pull/857#discussion_r277124603) [](ancestors = 277124603)